src/lib/message-router.js

```diff
@@ -132,7 +134,8 @@ MessageRouter.prototype.receiveRoutes = function * (payload, sender) {
     // We received a route from another connector, but that connector
     // doesn't actually belong to the route, so ignore it.
     if (route.source_account !== sender) continue
-    if (this.routingTables.addRoute(route)) gotNewRoute = true
+    const isPeerRoute = route.destination_ledger.startsWith(PEER_LEDGER_PREFIX)
+    if (this.routingTables.addRoute(route) && !isPeerRoute) gotNewRoute = true
```

Comment:
If RouteBroadcaster isn't going to broadcast the peer routes anyway, why is this needed?